### DISCUSSION TITLE
Unsupported MySQL version on import

### DISCUSSION THREAD

**Discussion by kirkouimet** [planetscale/discussion#302] [Documentation] [answered]
I am getting this error when connecting to my database to use the beta import feature:

`unsupported MySQL version detected: "5.5.31-0+wheezy1"`

Can someone from PlanetScale share a list of compatible databases? I can upgrade/change the database I am using on my side in order to facilitate the import.

**Comment by Phanatic** [comment, marked as answer]
Hi @kirkouimet,
Thanks for trying out PlanetScale's import feature. 
PlanetScale's support for mysql versions is documented here https://planetscale.com/docs/reference/mysql-compatibility#overview
